ghga_connector/cli.py

```diff
@@ -94,32 +94,47 @@ def download(  # pylint: disable=too-many-arguments
     output_dir: Path = typer.Option(
         ..., help="The directory to put the downloaded files into"
     ),
-    pubkey_path: Path = typer.Argument(
+    submitter_pubkey_path: Path = typer.Argument(
         "./key.pub",
-        help="The path to a public key from the key pair that will be used to encrypt the "
+        help="The path to a public key from the key pair that was announced in the "
+        + "metadata. Defaults to the file key.pub in the current folder.",
+    ),
+    submitter_private_key_path: Path = typer.Argument(
+        "./key.sec",
+        help="The path to a private key from the key pair that will be used to encrypt the "
         + "crypt4gh envelope. Defaults to the file key.pub in the current folder.",
     ),
 ):
     """
-    Command to download a file
+    Command to download files
     """
     core.RequestsSession.configure(CONFIG.max_retries)
     message_display = CLIMessageDisplay()
 
-    if not os.path.isfile(pubkey_path):
-        message_display.failure(f"The file {pubkey_path} does not exist.")
-        raise core.exceptions.PubKeyFileDoesNotExistError(pubkey_path=pubkey_path)
+    if not os.path.isfile(submitter_pubkey_path):
```

Comment:
submitter_pubkey_path is a Path object and supports .is_file(), no need for os here